spec/unit/middleware/rate_limiter_v2_api_spec.rb

```diff
@@ -68,6 +68,25 @@ module Middleware
             _, response_headers, _ = middleware.call(user_2_env)
             expect(response_headers['X-RateLimit-Remaining-V2-API']).to eq('80')
           end
+
+          context 'the user is exempt from v2 rate limiting' do
+            before do
+              allow(VCAP::CloudController::SecurityContext).to receive(:v2_rate_limit_exempted?).and_return(true)
+            end
+
+            let(:fake_request) { instance_double(ActionDispatch::Request, fullpath: '/v2/apps') }
+
+            it "doesn't decrease the count" do
+              allow(ActionDispatch::Request).to receive(:new).and_return(fake_request)
```

Comment:
This mock (and `fake_request`) seem unnecessary. I would propose to simply use `user_1_env`.